src/js/view/controls/controls.js

```diff
@@ -269,12 +272,16 @@ export default class Controls {
                     api.playToggle(reasonInteraction());
                     break;
                 case 37: // left-arrow, if not adMode and settings menu is hidden
-                    if (!adMode && menuHidden) {
```

Comment:
Is this a pattern we use in `controls`? Why not intercept in the slider element to handle custom keyboard behavior? adding it to `ui` in the slider will deal with focus automatically and you don't have to add new code and queries here.